x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/job/persistence/JobProvider.java

```diff
@@ -1135,6 +1137,9 @@ public void getForecastStats(String jobId, Consumer<ForecastStats> handler, Cons
                 .field(ForecastRequestStats.PROCESSED_RECORD_COUNT.getPreferredName()));
         sourceBuilder.aggregation(
                 AggregationBuilders.stats(ForecastStats.Fields.RUNTIME).field(ForecastRequestStats.PROCESSING_TIME_MS.getPreferredName()));
+        Script durationScript = new Script("doc['" + ForecastRequestStats.END_TIME + "'].value.getMillis() - doc['"
```

Comment:
It would be nice to have test coverage of this. There is a `JobProviderIT` class could you index some forecast stats in that and assert this method.